csharp/ql/integration-tests/all-platforms/cshtml_standalone/Files.expected

```diff
@@ -0,0 +1,3 @@
+| Program.cs |
+| Views/Home/Index.cshtml |
+| _semmle_code_target_codeql_csharp_integration_tests_ql_csharp_ql_integration_tests_all_platforms_cshtml_standalone_Views_Home_Index_cshtml.g.cs |
```

Comment:
Do we get this long name because this is the name of the integration test folder (+ Views/Home/Index.cshtml)?

Reply:
Yes.